corehq/apps/app_manager/id_strings.py

```diff
@@ -202,6 +202,37 @@ def media_resource(multimedia_id, name):
     return u'media-{id}-{name}'.format(id=multimedia_id, name=name)
 
 
+def module_icon_locale(module):
+    return u"modules.m{module.id}.icon".format(module=module)
+
+
+def module_audio_locale(module):
+    return u"modules.m{module.id}.audio".format(module=module)
+
+
+def form_icon_locale(form):
+    return u"forms.m{module.id}f{form.id}.icon".format(
+        module=form.get_module(),
+        form=form
+    )
+
+
+# ToDo - figure out whether media-strings are custom or not
```

Comment:
hey @snopoke what are 'non "app-string" id strings'? or should all of these be decorated with `pattern` making them custom app strings?